lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -61,17 +61,25 @@ def self.auto_generate_changelog(repo_name, github_token, rate_limit_sleep)
                                                  api_token: github_token)
           body = JSON.parse(pr_resp[:body])
           items = body["items"]
-
           if items.size == 1
             item = items.first
             message = "#{item['title']} (##{item['number']})"
             username = item["user"]["login"]
+            types_of_change = item["labels"]
+                              .map { |label_info| label_info["name"] }
+                              .select { |label| supported_types.include?(label) }
+                              .uniq
```

Comment:
same very slight optimization here, replacing `.uniq` with `to_set` will also make ever-so-slightly faster to check inclusion in `get_section_depending_on_types_of_change`
```suggestion
            types_of_change = item["labels"]
                              .map { |label_info| label_info["name"] }
                              .select { |label| supported_types.include?(label) }
                              .to_set
```